spec/index.bs

```diff
@@ -2282,6 +2282,29 @@ dictionary GPUImageBitmapCopyView {
 
   * {{GPUImageBitmapCopyView/origin}}: If unspecified, defaults to `[0, 0]`.
 
+### {{GPUCommandEncoder/copyBufferToBuffer()|GPUCommandEncoder.copyBufferToBuffer(GPUBuffer, GPUSize64, GPUBuffer, GPUSize64, GPUSize64)}} ### {#GPUCommandEncoder-copyBufferToBuffer}
```

Comment:
nit: Would you mind tweaking the formatting of this section to be closer to [requestDevice](https://gpuweb.github.io/gpuweb//#dom-gpuadapter-requestdevice) which I edited a few weeks ago? (I still haven't gotten to editing the rest of the spec to match, but will do so. Sorry that it's confusing right now.)

Mainly I'd just like to have:
- the easier-to-read section header formatting
- section header contains the definition
- arguments and return value
- validation has its own algorithm block (no need to reword it now, I'll get to that later; just put a block around it)

Reply:
Done